owncloudApp/src/main/java/com/owncloud/android/presentation/documentsprovider/cursors/FileCursor.kt

```diff
@@ -55,10 +54,8 @@ class FileCursor(projection: Array<String>?) : MatrixCursor(projection ?: DEFAUL
             flags = flags or Document.FLAG_DIR_SUPPORTS_CREATE
         }
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
-            flags = flags or Document.FLAG_SUPPORTS_COPY
-            flags = flags or Document.FLAG_SUPPORTS_MOVE
-        }
+        flags = flags or Document.FLAG_SUPPORTS_COPY
+        flags = flags or Document.FLAG_SUPPORTS_MOVE
```

Comment:
You can put this all together with the flags that don't require condition checks, the ones for delete and rename